server/static/js/chaos.js

```diff
@@ -144,8 +144,7 @@ function getText() {
     const result = document.getElementById("result");
     // read text from URL location
     const request = new XMLHttpRequest();
-    // request.open("GET", "voters?amount=20", true);
-    request.open("GET", "voters.json", true);
+    request.open("GET", "api/voters?amount=20", true);
```

Comment:
Supposedly fixed in #515, but doesn't look like it. So I'll revert back to the `json`s.